src/components/icon/icon.tsx

```diff
@@ -111,7 +111,8 @@ const typeToPathMap = {
   crossClusterReplicationApp: 'app_cross_cluster_replication',
   crosshairs: 'crosshairs',
   crossInCircleEmpty: 'crossInCircleEmpty',
-  crossInACircleFilled: 'crossInACircleFilled',
+  crossInACircleFilled: 'crossInCircleFilled',
+  crossInCircleFilled: 'crossInCircleFilled',
```

Comment:
nit - keys should be in alphabetical order. This mapping is the important bit that tells us which asset to use for a given type.

Reply:
Isn't the key already ordered in alphabetical order?

Reply:
I think it should be crossInACircleFilled, crossInCircleEmpty, crossInCircleFilled - it's L113 that needs to go in between.